test/interface/jump.jl

```diff
@@ -0,0 +1,3 @@
+function test_jump(Optimizer::Type{<:AbstractSampler})
+    
```

Comment:
Is this supposed to be empty?

Reply:
Not actually. This is hanging while the interface standards are not fully defined, I guess...